sdks/uniswapx-sdk/src/utils/OrderQuoter.ts

```diff
@@ -265,12 +280,27 @@ export class UniswapXOrderQuoter
       return [order.order.serialize(), order.signature];
     });
 
+    // const orderParser = new UniswapXOrderParser();
+    // const priorityOrders = orders.filter((o) => orderParser.getOrderTypeFromEncoded(o.order.serialize(), o.order.chainId) === OrderType.Priority);
```

Comment:
this is commented out for testing, because the integration tests don't deploy reactors via create2 and this will throw since the reactor address is technically unknown